packages/edit-site/src/components/actions/trash-post.js

```diff
@@ -17,6 +23,11 @@ export default function useMoveToTrashAction() {
 		() => ( {
 			id: 'move-to-trash',
 			label: __( 'Move to Trash' ),
+			type: ACTION_TYPES.primary,
+			icon: trash,
+			isEligible( { status } ) {
```

Comment:
This made me think of this interaction in wp-admin:

https://github.com/WordPress/gutenberg/assets/583546/7ee9a7a3-ea07-4fa6-a30f-d14815ea7485

Perhaps we should add `deletePermanently` action and make it available for every page (just as secondary)?

`deletePermanently` would probably require a different interaction (modal to confirm). It could be valuable to showcase what actions can do.

Reply:
Exactly! I didn't add any of those for now, because we can start adding actions incrementally, after we land the main PR.